vignettes/Pseudobulking-and-Filtered-Contrasts.Rmd

```diff
@@ -350,8 +350,8 @@ First, let's reuse the results of `FilterPseudobulkingContrasts` and apply some
 ```{r filter DEGs for heatmap}
 #bind the list into a dataframe using thresholds to determine the differentially expressed genes
 DEGs <- dplyr::bind_rows(DE_results) %>% 
-  dplyr::filter(abs(logFC) > 1) %>% 
-  dplyr::filter(FDR < 0.05)
+  dplyr::filter(abs(logFC) > 0.5) %>% 
+  dplyr::filter(FDR < 0.4) #0.4 used only for this vignette. One should generally use 0.05 to be consistent with field standards
```

Comment:
I did have to make the vignette a little more "fragile" in terms of what folks need to change if copy+pasting  since this is all fabricated, but I could just as easily swap the upstream filtering back to FALSE.  open to suggestions but I don't think it matters much.